src/Microsoft.PowerShell.Commands.Utility/commands/utility/UtilityCommon.cs

```diff
@@ -206,6 +206,58 @@ private set
         /// </summary>
         public string Path { get; private set; }
 
+        /// <summary>
+        /// Gets the hexadecimal representation of the <see cref="Offset64"/> value.
+        /// </summary>
+        public string HexOffset { get => string.Format("{0:X16}", Offset64); }
```

Comment:
Please use explicit culture parameter.

Reply:
Ah, of course. Using CurrentCulture?

Reply:
Yes, for user output we use CurrentCulture in formatting system.